test/metadata.test.js

```diff
@@ -40,7 +40,7 @@ test('child loggers works', async ({ ok, same, is }) => {
       is(child, this.lastLogger)
       is(30, this.lastLevel)
       is('a msg', this.lastMsg)
-      same(this.lastObj, { from: 'child', msg: 'a msg' })
+      same(this.lastObj, { from: 'child' })
```

Comment:
I have a feeling this might be a semver-major change.

Reply:
I agree, should I update the version in the package.json?